### DISCUSSION TITLE
Is Framer Motion required for chakra?

### DISCUSSION THREAD

**Discussion by HommeSauvage** [chakra-ui/chakra-ui#5485] [Q&A]
When analyzing my bundle, I found that Framer Motion is the largest package (337kb). I don't use it all in the project. Is it safe to remove it from a project using chakra ui?

Does Chakra rely on motion for the modal, menu and some small other animations?

**Comment by nayaabkhan** [comment]
We're trying to evaluate Chakra as well and wondering the same. I'm guessing you too had this question because you saw `framer-motion` in the [installation instructions](https://chakra-ui.com/docs/getting-started#installation). Even if you don't install it explicitly in your app's `package.json`, it will be installed as it is a dependency of `@chakra-ui/react` and a few components.

```
❯ npm list framer-motion
chakra@ /Users/nayaabkhan/Projects/github.com/nayaabkhan/learning/chakra
└─┬ @chakra-ui/react@1.8.1
  ├─┬ @chakra-ui/checkbox@1.6.3
  │ └── framer-motion@5.6.0 deduped
  ├─┬ @chakra-ui/menu@1.8.4
  │ └── framer-motion@5.6.0 deduped
  ├─┬ @chakra-ui/modal@1.10.5
  │ └── framer-motion@5.6.0 deduped
  ├─┬ @chakra-ui/popover@1.11.2
  │ └── framer-motion@5.6.0 deduped
  ├─┬ @chakra-ui/toast@1.5.2
  │ └── framer-motion@5.6.0 deduped
  ├─┬ @chakra-ui/tooltip@1.4.4
  │ └── framer-motion@5.6.0 deduped
  ├─┬ @chakra-ui/transition@1.4.3
  │ └── framer-motion@5.6.0 deduped
  └── framer-motion@5.6.0
```

**Comment by LuisValgoi** [comment]
I don't want to have framer-motion on my bundle size when using ChakraUI... 

Is this something that I will have to live on
or....
Does anyone have any idea on how to decrease the bundle size when using chakraui?

**Reply by ivan-kleshnin** [reply]
No way to decrease it man. TailwindCSS f.t.w 👇 

![01 nextjs-bundlesize](https://user-images.githubusercontent.com/2128182/204268828-ee622f14-6592-4b3b-a187-dcc34f874f79.jpg)
![03 gtmetrix-summary](https://user-images.githubusercontent.com/2128182/204268854-c3262714-c861-41cb-81e6-f2b42a89497f.jpg)

**Comment by IRediTOTO** [comment]
Same question here. Can we remove it, it's so heavy

**Comment by stark-bit** [comment]
Unable to use ChakraUi because of import error from `framer-motion`
```
Can't import the named export 'AnimatePresence' from non EcmaScript module (only default export is available)
```
The error can be resolved by upgrading some framer-motion dependancies but then there's 200+ new errors to deal with.
It's not the first project that I had to scrap chakraUI due to dependancy issues.

So yeah, i'd like to be able to use ChakraUi without framer-motion pls.

**Comment by campmedia** [comment]
I started a Poll https://github.com/chakra-ui/chakra-ui/discussions/8377
